google/cloud/bigtable/internal/defaults_test.cc

```diff
@@ -392,6 +397,15 @@ TEST(ConnectionRefreshRange, BothSetInvalidUsesMax) {
             absl::FromChrono(opts.get<MaxConnectionRefreshOption>()));
 }
 
+TEST(ConnectionRefreshRange, DisabledIfEmulator) {
+  ScopedEnvironment emulator("BIGTABLE_EMULATOR_HOST", "emulator-host:8000");
+  auto opts = DefaultOptions();
+
+  // Zero duration means connection refreshing is disabled.
+  EXPECT_EQ(absl::ZeroDuration(),
+            absl::FromChrono(opts.get<MaxConnectionRefreshOption>()));
```

Comment:
Seems odd to convert these to `abls::Duration` just to compare to zero.  Maybe:

https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/testing_util/chrono_output.h

?

Reply:
Done, TIL (or maybe TIR)